py/janitor/site/templates/layout.html

```diff
@@ -23,17 +23,23 @@
         <script src="/_static/chart.js"></script>
         <script src="/_static/janitor.js"></script>
         {% include "analytics.html" ignore missing without context %}
-        <div class="document">
-            {% block sidebar %}
-            {% endblock sidebar %}
+        {% if layout != 'plain' %}
+        {# djlint:off H025 #}<div class="document">{# djlint:on #}
```

Comment:
No need for overrides here, and having an uneven number of opens/closes also makes this hard to read.

Reply:
I was trying to reduce the amount of duplicate code. 
Will alter formatting then and drop `djlint`